libraries/botbuilder-middleware-aws-comprehend/src/engine.ts

```diff
@@ -0,0 +1,128 @@
+import { Engine } from '@botbuildercommunity/middleware-engine-core';
+import * as aws from 'aws-sdk';
+
+/**
+ * @module botbuildercommunity/middleware-aws-comprehend
+ */
+
+export class AWSComprehendEngine extends Engine {
+    public client: aws.Comprehend;
+    public constructor() {
+        super();
+        this.client = new aws.Comprehend({ apiVersion: '2017-11-27', region: 'us-east-1' });
```

Comment:
Is there only one region? I believe that region should be in the config you pass to the middleware.

Reply:
Changed.